builder/repo_control.go

```diff
@@ -36,7 +36,7 @@ type dnfRepoConf struct {
 }
 
 type repoInfo struct {
-	cacheDir  string
+	cacheDirs map[string]string
```

Comment:
We are throwing away the value, so cacheDirs could be `map[string]bool` (typically done in Mixer) or `map[string]struct{}`